spec/services/manifest_builder_v3/by_resource_type/manifest_builder_v3_audio_resource_spec.rb

```diff
@@ -49,14 +51,36 @@
       output = manifest_builder.build
       # pres 3 context is always an array
       expect(output["@context"]).to include "http://iiif.io/api/presentation/3/context.json"
-      # logo is always an array
-      expect(output["logo"].first).to include("id" => "https://www.example.com/pul_logo_icon.png")
+      # logo is part of a provider object
+      provider = output["provider"][0]
+      expect(provider["type"]).to eq "Agent"
+      expect(provider["logo"].first).to include("id" => "https://www.example.com/pul_logo_icon.png")
       # Logical structure should be able to have nested and un-nested members.
       expect(output["structures"][0]["items"][0]["id"]).to include "#t="
       expect(output["structures"][1]["items"][0]["items"][0]["id"]).to include "#t="
       expect(output["behavior"]).to eq ["auto-advance"]
       # downloading is blocked
-      expect(output["service"][0]).to eq({ "@context" => "http://universalviewer.io/context.json", "profile" => "http://universalviewer.io/ui-extensions-profile", "disableUI" => ["mediaDownload"] })
+      expect(output["items"][0].dig("rendering", 0, "format")).to be_nil
+      # Use the Audio type
+      expect(output["items"][0]["items"][0]["items"][0]["body"]["type"]).to eq "Sound"
+
+      # Validate manifest
+      expect(JSON::Validator.fully_validate(schema, output)).to be_empty
+    end
+
+    context "with an accompanying image file" do
+      let(:image_file) { fixture_file_upload("files/example.tif") }
+      let(:change_set) { ScannedResourceChangeSet.new(scanned_resource, files: [file, image_file], downloadable: "none") }
+
+      it "builds a manifest with an accompanyingCanvas element", run_real_characterization: true do
+        change_set_persister.save(change_set: change_set)
+        output = manifest_builder.build
+
+        expect(output["items"][0]["accompanyingCanvas"]["width"]).to eq 200
```

Comment:
Does this accompanyingCanvas render in UV?

Reply:
Hmm... It doesn't appear to in this version of UV. I guess we could add the posterCanvas back until we switch over to Clover to audio and video.

Reply:
Added back posterCanvas.